plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -36,8 +36,9 @@ FileSystemAdaptor.prototype.isReady = function() {
 
 FileSystemAdaptor.prototype.getTiddlerInfo = function(tiddler) {
 	//Returns the existing fileInfo for the tiddler. To regenerate, call getTiddlerFileInfo().
+	var self = this;
 	var title = tiddler.fields.title;
-	return this.boot.files[title];
+	return self.boot.files[title];
```

Comment:
hmmm, Doesn't make too much sense for me. IMO `this.` is good enough.

Reply:
In [line 20](https://github.com/Jermolene/TiddlyWiki5/pull/5415/files#diff-ccebed3949d890544181387dc9b9cde2a9f2f81870103a6de069f480ccca97abL20) we should remove `var self = this`  since the variable isn't used and only wastes time and memory

Reply:
**Did you add the `self.xxx` .. for consistency, to make it easier to copy/paste code??**

Reply:
Thank you. I think I was trying to catch an accidental "replace all". Agree.